lib/phlex/elements.rb

```diff
@@ -32,6 +32,8 @@ def #{element}(**attributes, &block)
 			end
 
 			alias_method :_#{element}, :#{element}
+			alias_method :__phlex_#{element}__, :#{element}
```

Comment:
One very minor performance thought I had here: if we changed the element method definition around so that `__phlex_#{element}__` was the original method, and `#{element}` was the alias, then we could check if a method is the real Phlex element method by checking `UnboundedMethod#original_name` instead of comparing two `UnboundedMethod` instances. That would reduce the amount of UnboundedMethod allocations when checking to see if something can be optimized.

Reply:
This would be even better 👍